test/integration/submission/migration_utils/static-schema-data.ts

```diff
@@ -0,0 +1,4566 @@
+// Legacy, fixed Schema Versions from the stub-schema
+
+// The first version of the schema, but using object notation for schemas and fields for easy modification
+export const schemaV1_accessible = {
```

Comment:
what does `accessible` mean here?

Reply:
it uses object notation, where it uses keys for the file names and the fields. This makes it easier to extend upon the schema when trying to make a new version to migrate to, so you can easily modify the properties you want to change.

 Original schemas have them in arrays, so that would require knowledge of the array index

Reply:
did you build this manually ?

Reply:
I used a local script to generate it. I didn't keep the code but it used a lodash function.

Reply:
why not keep that script and generate on the fly ?

Reply:
I believe I had initially tried that but typescript produced errors when I tried to access the keys of the object since there was no type/interface defined for it

Reply:
you can use `any`

Reply:
but we already have alot of Types already defined in clinical service too that can be used

Reply:
this can be harder to maintain on the long run, that's why I'm asking about this

Reply:
I used `any` but then lost intellisense on the suggestion property and field names. I assumed version_1 would remain static but you are correct that it should be easily extendible. I think the best thing for me to do then is to fully define the type.

Reply:
I can help you in this tomorrow to figure out the best way without alot of manual typing just remind me if I forget please